views/components/common/CertIcon.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import styled from 'styled-components/macro'
```

Comment:
this one should be in  views/components/certification/ not common

Reply:
Updated